build.py

```diff
@@ -185,10 +185,21 @@
             "test_requirements.txt",
         ]
         subprocess.run(pip_install_args, check=True, text=True, cwd=pip_src)
-        pip_install_args = [python_bin, "-m", "pip", "install", "-e", "."]
+        pip_install_args = [
+            python_bin,
+            "-m",
+            "pip",
+            "install",
+            "--force-reinstall",
+            "--no-index",
+            "--find-links=" + wheels_dir,
```

Comment:
This will stop working when the wheel is published.

https://pip.pypa.io/en/stable/cli/pip_install/#finding-packages

The path to the wheel file needs to be used:
```
pip install --force-reinstall ./path/to/package.whl
```

Reply:
Hmmm. I think I'll take a look again tomorrow. The problem is that I can't figure out how to get the full path to the .whl, since the filename is generated based on the version/platform. `--no-index` is supposed to ignore PyPi but at this point I don't trust my understanding.

Reply:
For the time being is it possible to use the wildcards instead of exact file names? E.g., "qsharp_python-*.*.*-cp37-abi3-*.whl".
Sometime later we plan to have a single tool to specify/update the version synchronously in all the .toml, .yml, etc. (likely including this "build.py").